management/src/commands/send-two-fa-verification-code.ts

```diff
@@ -161,10 +161,16 @@ async function sendTwoFAVerificationCode(
     localeCodeOverride,
   }: SendTwoFAVerificationCodeInput,
 ): Promise<SendTwoFAVerificationCodeResult> {
-  const twoFAfunction =
-    identity.type.toLocaleLowerCase() === "phone"
-      ? sendPhoneTwoFAVerificationCode
-      : sendEmailTwoFAVerificationCode;
+  const twoFAfunction = ((): TwoFAFunction => {
+    switch (identity.type.toLocaleLowerCase()) {
+      case "phone":
+        return sendPhoneTwoFAVerificationCode;
+      case "email":
+        return sendEmailTwoFAVerificationCode;
+      default:
+        throw new InvalidIdentityTypeError();
```

Comment:
This way we are sure that only `email` and `phone` identity types are ok. All the other would throw `InvalidIdentityTypeError`